client/package.json

```diff
@@ -2,11 +2,9 @@
   "name": "@concord-consortium/token-service",
   "version": "1.2.0",
   "description": "Provides a CRUD api to create resources and an api endpoint to generate AWS credentials from those resources.",
-  "main": "lib/client.js",
+  "main": "lib/client/src/client.js",
```

Comment:
Should this be `"lib/client/client.js" - it seems weird that we would have a `src` sub-folder in the output folder.